blog/2023-02-28-integrate-pikapool/index.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: integrate-pikapool
+title: Integrate Pikapool in 1, 2, 3
+authors: [arceus, viola]
+tags: [technical]
+---
+
+Conducting an NFT sale that **maximises revenue for creators** and **provides the best experience for bidders** is a non-trivial, difficult problem requiring weeks of game theory research and engineering resources...Is what was the case prior to the launch of Pikapool! ⚡
+
+Pikapool can be integrated into a project in **three easy steps** and just a few lines of code.
+
+## Step 1 - Introduce yourself
+
+Introduce yourself in [our Discord server](https://discord.gg/J6ypfSe5GW) where our team will take note of your requirements and provide end-to-end support for your sale.
+
+## Step 2 - Become Pikapatible
+
+An NFT contract can be made compatible with Pikapool with just a few trivial code changes.
+
+![Pikapadible](./pikapatible.png)
```

Comment:
Looks great, I think it's good to merge with the spelling error on line 20 fixed!